x-pack/plugins/lens/public/datatable_visualization/visualization.tsx

```diff
@@ -143,15 +143,28 @@ export const datatableVisualization: Visualization<DatatableVisualizationState>
       groups: [
         {
           groupId: 'columns',
-          groupLabel: i18n.translate('xpack.lens.datatable.columns', {
-            defaultMessage: 'Columns',
+          groupLabel: i18n.translate('xpack.lens.datatable.breakdown', {
+            defaultMessage: 'Break down by',
           }),
           layerId: state.layers[0].layerId,
-          accessors: sortedColumns,
+          accessors: sortedColumns.filter((c) => datasource.getOperationForColumnId(c)?.isBucketed),
           supportsMoreColumns: true,
-          filterOperations: () => true,
+          filterOperations: (op) => op.isBucketed,
           dataTestSubj: 'lnsDatatable_column',
         },
+        {
+          groupId: 'metrics',
+          groupLabel: i18n.translate('xpack.lens.datatable.metrics', {
+            defaultMessage: 'Metrics',
+          }),
+          layerId: state.layers[0].layerId,
+          accessors: sortedColumns.filter(
+            (c) => !datasource.getOperationForColumnId(c)?.isBucketed
+          ),
+          supportsMoreColumns: true,
+          filterOperations: (op) => !op.isBucketed,
+          dataTestSubj: 'lnsDatatable_metrics',
+        },
```

Comment:
I like this change a lot! That's maybe outside of the scope of this PR, but it can be a perfect oportunity to show that metric is required:
<img src="https://user-images.githubusercontent.com/4283304/91279326-a1d55200-e785-11ea-8034-c2c721467775.png" data-canonical-src="https://user-images.githubusercontent.com/4283304/91279326-a1d55200-e785-11ea-8034-c2c721467775.png" height="300"  />
just like here:
![image](https://user-images.githubusercontent.com/4283304/91279405-ba456c80-e785-11ea-9a27-a011b566c0fd.png)

Reply:
I agree we should do this. I personally don't think it's out of scope of this PR since this PR **creates** the new section. I'd make it a blocker for this PR.

Reply:
@cchaos At the time you made this comment, it was already required. This is an older commit.

Reply:
Great! There wasn't any response to this comment, so I couldn't be sure.